styles.css

```diff
@@ -360,3 +384,155 @@ textarea {
   background: #0d151e;
   border-radius: 10px;
 }
+
+.about_btn:hover {
+  border-radius: 10px 20px 10px 20px;
+}
+
+/* Media Queries */
+@media (min-width: 768px) {
+  header {
+    padding: 16px 143px;
+  }
+
+  div.hamburger {
+    display: none;
+  }
+
+  ul.nav_bar {
+    display: flex;
+    padding-left: 0;
+    list-style-type: none;
+  }
+
+  ul.nav_bar li {
+    padding: 0 12px;
+  }
+
+  ul.nav_bar .nav_links {
+    text-decoration: none;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  #headline {
+    background: url('/images/Header\ bg.svg'), #fff;
+
+    /* padding: 289px 260px; */
+    padding: 10rem 5rem;
+    background-size: 100vw 100vh;
+    background-position: center;
+    border-bottom-left-radius: 200px;
+    background-repeat: no-repeat;
+  }
+
+  div.works_details p {
+    font-size: 16px;
+  }
+
+  #headline h1 {
+    font-size: 48px;
+    line-height: 60px;
+  }
+
+  #headline p {
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  ul.works_grid {
+    padding: 5rem;
+  }
```

Comment:
- [ ] **[OPTIONAL]:** You did a great job applying the suggestions from the last reviewer! 💯 However, there's still room to improve regarding the responsiveness of your page. Please avoid using fixed units to center/position items. I strongly suggest that you delete this padding and, instead, use the `justify-items: center;` property to center your elements in the grid. Take in mind that in order to make this work, you should specify a `width` for the child elements (Something like 80%). 🤓 🛠️ 

You can read more about this property in the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-items).

Reply:
Thanks so much for this feedback. It really addressed one of my major concerns as regards this project.